benchalerts/benchalerts/conbench_dataclasses.py

```diff
@@ -30,26 +30,24 @@ class RunComparisonInfo:
     contender_info
         The dict returned from Conbench when hitting /runs/{contender_run_id}. Contains
         info about the run's ID, commit, errors, links, etc.
-    baseline_info
-        The dict returned from Conbench when hitting /runs/{baseline_run_id}, if a
-        baseline run exists for this contender run. Contains info about the run's ID,
-        commit, errors, links, etc.
+    baseline_run_type
+        The user-given baseline run type to look for.
     compare_results
         The list returned from Conbench when hitting
         /compare/runs/{baseline_run_id}...{contender_run_id}, only if a baseline run
         exists for this contender run. Contains a comparison for every benchmark result
         to its baseline, including the statistics and regression analysis.
     benchmark_results
         The list returned from Conbench when hitting
-        /benchmarks?run_id={contender_run_id}, only if the contender run has errors and
-        there is no baseline run. Contains info about each benchmark result in the
-        contender run, including statistics and tracebacks. Only used when a baseline
-        run doesn't exist because otherwise all this information is already in the
-        compare_results.
+        /benchmark-results/?run_id={contender_run_id}, only if the contender run has
+        errors and there is no baseline run. Contains info about each benchmark result
+        in the contender run, including statistics and tracebacks. Only used when a
+        baseline run doesn't exist because otherwise all this information is already in
+        the compare_results.
     """
 
     contender_info: dict
-    baseline_info: Optional[dict] = None
+    baseline_run_type: str
```

Comment:
Should this use the enum?

Reply:
There would be a circular dependency if we did so. I think it's okay to use a string since this dataclass is entirely internal. The only time this dataclass is ever constructed is here:
https://github.com/conbench/conbench/blob/a12c8e62708f668ad293712ff4389bd9bd157c1f/benchalerts/benchalerts/pipeline_steps/conbench.py#L92-L95
So it's safe to let this be a string.